contributing.rst

```diff
@@ -345,7 +345,7 @@ Run ``git remote -v`` to check the status, you should see something like this:
 Install Dependencies
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-The first time you clone down the repo, you'll need to install the dependencies. Make sure you have your Python 3 virtual environment set up and activated and then:
+The first time you clone down the repo, you'll need to install the dependencies. Make sure you have your Python 3 virtual environment set up and activated from the docs directory and then:
```

Comment:
That is not actually what we recommend.

Reply:
This one I've changed also because of tests I've performed while setting up my local environment. 

From reading the documentation I understood that I should activate the virtual environment where it was created so I left the recently cloned *docs* directory and went to the previously created *odkenv* directory and after I activated the virtual environment and ran the command to install the dependencies I got the error message that the file requirements.txt didn't exist.

So I left the *odkenv* directory to the *odk* folder that has both the *odkenv* and the *docs* directories and tried again, got the same message.

My last try was: activate the virtual environment from within the *docs* directory with the code `$ source ../odkenv/bin/activate` and then run the command to install the dependencies, and that worked, no error messages. 

That is why I felt that this information should be in the text.

Reply:
It doesn't matter where you activate the environment (thought that obviously affects the specific path you have to type).
The issue is that you have to be in the docs repo to `pip install` from the requirements file

Reply:
So maybe change the phrasing of what I added to the text to:
> 
...virtual environment set up and activated in the docs repo and then:

Would that be more accurate?

Also (sorry, git is new to me), do the cloned _**docs**_ repo resides just inside the _**docs**_ directory in my local file system? Or should I be able to access it from the _**odk**_ directory or even from the _**odkenv**_ one?